
int32 pr_serial_size
int32 no_serial_size
int32 ls_serial_size
int32 tf_serial_size
int32 ra_serial_size
int32 od_serial_size
int32 ma_serial_size
